src/text-expander-element.ts

```diff
@@ -189,8 +189,10 @@ class TextExpander {
   onKeydown(event: KeyboardEvent) {
     if (event.key !== 'Escape') return
     this.deactivate()
-    event.stopImmediatePropagation()
-    event.preventDefault()
+    if (this.menu || this.combobox) {
```

Comment:
`.diactivate()` will set menu to null. 
https://github.com/github/text-expander-element/blob/ef9bd652ae8294a2909cd2b8d79f2c32d6a73852/src/text-expander-element.ts#L84-L87

Maybe we should put if after this `if`?

Reply:
yeah :( i was messy and missed it

Reply:
i moved up the statement